summary-object.go

```diff
@@ -100,6 +100,18 @@ type ObjectPartSummary struct {
 	Size     int64
 }
 
+// ObjectUnknownSummary is returned from minio when calling ObjectSummary.
+// This struct contains information about files that are not part of any object structure.
+type ObjectUnknownSummary struct {
+	Pool     int
+	Host     string
+	Set      int
+	Drive    string
+	Filename string
+	Size     int64
```

Comment:
```suggestion
	ModTime time.Time
	Size     int64
```
I guess we already have it?

Reply:
mod time is on the meta summary object, unless you want OS mode time for each part ?

Reply:
Mtime (File, not metadata) is actually quite useful, so see last write time was consistent. For abandoned files I can tell us when it was dumped there.